### PR TITLE
T173919 Implement english banner

### PR DESCRIPTION
**Pull request by gbirke**

### CODE REVIEW

File: desktop/templates/banner_text_var.hbs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (wiese) [inline desktop/templates/banner_text_var.hbs]: Is that good?

File: desktop/banner_ctrl.js [hunk 2]
```diff
@@ -118,7 +118,7 @@ function setupAmountEventHandling() {
 function validateAndSetPeriod() {
 	var selectedInterval = $( '#WMDE_Banner-frequency input[type=radio]:checked' ).val();
 	if ( typeof selectedInterval === 'undefined' ) {
-		BannerFunctions.showFrequencyError( 'Bitte wählen Sie zuerst ein Zahlungsintervall.' );
+		$( '#WMDE_Banner' ).trigger( 'validation:period:error', Translations[ 'no-interval-message' ] );
```
Comment (wiese) [inline desktop/banner_ctrl.js]: Here we change both, the translation (understandable) as well as the mechanism to deploy it - switching from dedicated function to pure `$` feels like a deterioration. Is this intentional?

**Review by wiese — CHANGES_REQUESTED** [review]
Mind the unresolved merge conflict!

File: desktop/banner_ctrl.js (hunk 2 above)
Comment (KaiNissen) [reply desktop/banner_ctrl.js]: Hm, I'm not sure whether it is better to hide `$` in the respective function. Might be a good idea for having the event handling in `BannerFunctions` only, though.

In my opinion, what happens in `BannerFunctions` should be dropped/extracted into modules over time anyway. In this case we should add an `EventManager` that cares for handling all events.

**Review by wiese — APPROVED** [review]

**Pull request merged**